joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/dependency/GradleDependencies.scala

```diff
@@ -102,114 +102,127 @@ object GradleDependencies {
     GradleInitScript(content, taskName)
   }
 
+  private def getGradleProjectInfo(projectDir: Path, projectName: String): Try[GradleProjectInfo] = {
```

Comment:
For this function i would also switch to `Option` as return value.

Reply:
:+1: